### PR TITLE
Bootstrap style

### PR DESCRIPTION
**Pull request by tsohleDev**

### CODE REVIEW

**Comment by Habtamu210** [comment]
# Comments:
- Fix linter errors
- Use bootstrap classes for the bootstrap section

**Comment by lawrahkonwea** [comment]
- The header section has no bootstrap designs to it
- The images fit into the Figma design

File: index.html [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+                    <li class="nav-item">
+                        <a class="nav-link active" aria-current="page" href="#about">About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link active" aria-current="page" href="#contact">Contact</a>
+                    </li>
+                </ul>
+            </div>
+            
+        </div>
```
Comment (Talha-Hanif5666) [inline index.html:48]: I think its better to not leave any empty line. Please fix this

**Review by Talha-Hanif5666 — COMMENTED** [review]
Hello @tsohleDev 
157397398-00904e26-60ec-46cd-9510-240661c250ef

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



**Highlights**
 Descriptive PR :heavy_check_mark:
Good commit messages :heavy_check_mark:

**Changes Required** :recycle: 
 Please kindly check the comments under the review point_down point_down
**Optional suggestions**
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

**Pull request merged**